stylesheets/commons/Faction.less

```diff
@@ -84,8 +85,11 @@ Author(s): hjpalpha
 	}
 
 	.warcraft-undead,
+	.stormgate-celestial,
 	.warcraft-undead th,
-	.warcraft-undead td {
+	.stormgate-celestial th,
+	.warcraft-undead td,
+	.stormgate-celestial td {
 		background: var( --clr-vividviolet-30, #200033 ) !important;
```

Comment:
Is important needed? Would it solve to prefix 'body' to it instead? Like `body .stormgate-celestial`?

Reply:
sadly yes, due to `.wikitable` already having `!important`